### DISCUSSION TITLE
is there any restriction for Iranian developers or requests that are made from Iran?

### DISCUSSION THREAD

**Discussion by yamin8000** [supabase/supabase#2614] [Questions]
Recently I'm experiencing some problems when I'm trying to make request to my API from devices based in Iran or I even can't access my table editor, I'm still not sure if it's a problem with supabase itself or restrictions from my ISP in Iran.
Given that supabase is based in USA and [USA imposed some export laws](https://github.blog/2021-01-05-advancing-developer-freedom-github-is-fully-available-in-iran/) that prevents USA companies to work with Iranian people so my question is that is there any restriction already or in the future?

**Comment by silentworks** [comment]
I don't know if this would be the case by Supabase themselves, but it might be the case of the company that hosts the infrastructure or as you stated it could be your ISP restricting USA based services. Maybe someone from the team could address this.
